jenkins/release.jenkinsFile

```diff
@@ -0,0 +1,299 @@
+lib = library(identifier: 'jenkins@5.6.1', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+def VERSION = ''
```

Comment:
I am curious as to the use of a global variable here. Is there not a way to pass the parameter from the trigger to the job directly?